extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -794,6 +794,13 @@ public enum ResponseMode {
         @ConfigItem(defaultValueDocumentation = "true")
         public Optional<Boolean> idTokenRequired = Optional.empty();
 
+        /**
+         * Internal ID token lifespan.
+         * This property is only checked when an internal IdToken is generated when Oauth2 providers do not return IdToken.
+         */
+        @ConfigItem(defaultValueDocumentation = "5M")
+        public Optional<Duration> internalIdTokenLifespan = Optional.empty();
```

Comment:
The default is `5M`?

Reply:
Hi @gastaldi, yes, I'm using a `Duration` format, please see `smallrye.jwt.new-token.lifespan` at https://quarkus.io/guides/security-jwt-build#smallrye-jwt-builder-configuration, `300 secs` is a default value.
`smallrye.jwt.new-token.lifespan` can actually be used right now for this case - but it is not a solution which will always work, if another token for other purposes has to be created then it may not work.
I've updated the tests to also check that configuring it in `applicaion.properties` works.

Does it clarify it ?